### PR TITLE
Menu onclick addition

### PR DESCRIPTION
**Pull request by dannon**
This adds functionality for specifying simple 'onclick' events in galaxy menu items.  This is very useful for displaying things in Galaxy without requiring an explicit webapp endpoint.

### CODE REVIEW

File: client/galaxy/scripts/galaxy.masthead.js [hunk 1]
```diff
@@ -297,66 +297,76 @@ var GalaxyMastheadTab = Backbone.View.extend({
...
+                e.preventDefault();
+                menuOptions.onclick();
+            });
+        } else {
+            $item.on('click', function(e){
+                // prevent default
+                e.preventDefault();
+
+                //self.options.click_fn()
                 return true;
```
Comment (nsoranzo) [inline client/galaxy/scripts/galaxy.masthead.js]: Either this is not correct or the following code will be never executed.

Comment (dannon) [inline client/galaxy/scripts/galaxy.masthead.js]: Good catch!  Debugging code I'd left in, one sec.

Comment (nsoranzo) [inline client/galaxy/scripts/galaxy.masthead.js]: Thanks! Looks good to me now.

**Comment by guerler** [comment]
:+1:

**Pull request merged**